src/app.js

```diff
@@ -25,6 +27,7 @@ app.use(favicon(__dirname + '/public/favicon.ico'));
 app.use('/api/v1/parents', parentRouter);
```

Comment:
this can be in your router.index

so that you can have something like: `app.use('/api/v1/', router);` here